docs/source/recipes/showImages.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+
+import numpy as np
+from PIL import Image
+from psychopy import core, event, visual
+from skimage import img_as_float
```

Comment:
@peircej I don't think we have `scikit` as a requirement. I think demos should work with the packages we ship with standalone. @tomwallis we may need to use builtin numpy and Pillow functions to convert the image to an array. Perhaps something like this ...

```
im = Image.open('myPic.png')  # Pillow
arr = numpy.array(im, order='C')  # convert to array, must be C contiguous for OpenGL 
arr /= 255.  # normalize
```

Reply:
Thanks for the suggestion @mdcutone. I removed the dependency on `skimage` and added a few more docs. I also thought it would make sense to add some more information to the docstring of `ImageStim` itself.